e2e/keywords/volume.resource

```diff
@@ -1,60 +1,34 @@
 *** Settings ***
-Documentation       Longhorn volume related keywords
-
-Library             ../libs/keywords/volume_keywords.py
-Resource            ./node.resource
+Documentation    Volume Keywords
 
+Library    ../libs/keywords/volume_keywords.py
 
 *** Keywords ***
-Create Volume With Fields
-    [Arguments]    ${list_of_fields}
-    ${volume_name}    ${manifest} =    Create Volume Base Manifest
+Create a volume ${size} GB with ${replica_count} replicas
+    ${volume_name} =    create_volume   ${size}    ${replica_count}
+    attach_volume    ${volume_name}
+    Set Test Variable    ${volume_name}
 
-    FOR    ${field}    IN    @{list_of_fields}
-        ${manifest} =    Update Manifest With Fields    ${manifest}    ${field}
-    END
+Write data to the volume
+    ${volume_data_checksum} =     write_volume_random_data   ${volume_name}    1024
```

Comment:
Yes, there should be another keyword can let users specify the write size later. But also we should not restrict users must specify the write size. Like in our existing test cases in `manager/integration/tests/`, there are lots of test cases just generally describe `write some data` in the test steps, since the write size isn't what they concerned in that test case.

Reply:
Exactly. Just make it configurable but also have a default value.